db/migrate/20180227160557_drop_action_count_columns_from_topics.rb

```diff
@@ -0,0 +1,9 @@
+class DropActionCountColumnsFromTopics < ActiveRecord::Migration[5.1]
```

Comment:
Oh ... we don't need a dummy migration here, just pick the last one currently, it will work fine. also keep in mind column dropper reruns

Reply:
I think the last migration would likely have been created 60 minutes ago and that would cause the columns to be dropped immediately.

Reply:
I mimicked what has been done in the past when columns were dropped. I think @tgxworld is right. Picking the last migration might lead to columns being dropped immediately. I'm not sure if we really want that to happen.